sql/moz-fx-data-shared-prod/firefox_desktop_derived/baseline_clients_last_seen_v1/backfill.yaml

```diff
@@ -0,0 +1,9 @@
+2025-02-10:
+  start_date: 2021-12-01
```

Comment:
just want to flag that `legacy_telemetry_client_id` and `windows_build_number` will not be present in the data for this month, so we won't be able to validate those metrics from this test.

Reply:
I'm assuming we can't test on a more recent month because of the dependency issue?

Reply:
> I'm assuming we can't test on a more recent month because of the dependency issue?

That is correct. I also wanted to get an estimate of what it takes to run for one month especially because the runs need to happen sequentially on a single thread